### DISCUSSION TITLE
PIA port becomes closed after a day or so, then I have to manually restart the container for it to fetch a new open port

### DISCUSSION THREAD

**Discussion by aspot** [haugene/docker-transmission-openvpn#2438] [Q&A]
Any way to automate this? Either to have a script to fetch a new open port from PIA every so often, or a script to stop and restart the docker image? Thanks.

Docs mention problem with losing connectivity (https://haugene.github.io/docker-transmission-openvpn/faq/#container_loses_connection_after_some_time), but I can still access Transmission. The problem is the port Transmission uses becomes closed and transfers stop.

**Comment by emk2203** [comment]
I have it run for days to weeks, port stays open. It's something in your setup.
